case_specification.tex

```diff
@@ -0,0 +1,300 @@
+		 Th230	 & 	 	& 	2.625E-09 & 	 	 & 	2.139E-09 \\ \hline 
+		 Th232	 & 	 	& 	4.174E-10 & 	 	 & 	4.425E-11 \\ \hline 
+		 Bi209	 & 	 	& 	6.607E-16 & 	 	 & 	2.600E-14 \\ \hline 
+		 Ac227	 & 	 	& 	3.096E-14 & 	 	 & 	4.840E-15 \\ \hline 
+		 Pa231	 & 	 	& 	9.246E-10 & 	 	 & 	1.300E-10 \\ \hline 
+		 U232	 & 	 	& 	0.000 & 	 	 & 	0.000 \\ \hline 
+		 U233	 & 	 	& 	2.213E-09 & 	 	 & 	5.528E-09 \\ \hline 
+		 U234	 & 	0.000& 	0.000 & 	 	 & 	0.000 \\ \hline 
+		 U235	 & 	0.032& 	0.007 & 	0.002 & 	0.000 \\ \hline 
+		 U236	 & 	 	& 	0.005 & 	 	 & 	0.000 \\ \hline 
+		 U238	 & 	0.968& 	0.920 & 	0.887 & 	0.808 \\ \hline 
+		 Np237	 & 	 	& 	0.000 & 	 	 & 	0.000 \\ \hline 
+		 Pu238	 & 	 	& 	0.000 & 	0.001 & 	0.001 \\ \hline 
+		 Pu239	 & 	 	& 	0.006 & 	0.060 & 	0.085 \\ \hline 
+		 Pu240	 & 	 	& 	0.002 & 	0.027 & 	0.027 \\ \hline 
+		 Pu241	 & 	 	& 	0.001 & 	0.014 & 	0.003 \\ \hline 
+		 Pu242	 & 	 	& 	0.000 & 	0.005 & 	0.001 \\ \hline 
+		 Pu244	 & 	 	& 	2.864E-08 & 1.508E-07 & 	5.461E-09 \\ \hline 
+		 Am241	 & 	 	& 	6.442E-05 & 	 	 & 	0.001 \\ \hline 
+		 Am242m	 & 	 	& 	8.533E-07 & 	 	 & 	7.961E-05 \\ \hline 
+		 Am243	 & 	 	& 	0.000 & 	 	 & 	0.000 \\ \hline 
+		 Cm242	 & 	 	& 	2.589E-05 & 	 	 & 	5.331E-05 \\ \hline 
+		 Cm243	 & 	 	& 	0.000 & 	 	 & 	3.242E-06 \\ \hline 
+		 Cm244	 & 	 	& 	8.561E-05 & 	 	 & 	0.000 \\ \hline 
+		 Cm245	 & 	 	& 	5.721E-06 & 	 	 & 	3.936E-05 \\ \hline 
+		 Cm246	 & 	 	& 	7.295E-07 & 	 	 & 	1.434E-05 \\ \hline 
+		 Cm247	 & 	 	& 	0.000 & 	 	 & 	5.317E-07 \\ \hline 
+		 Cm248	 & 	 	& 	7.691E-10 & 	 	 & 	0.000 \\ \hline 
+		 Cm250	 & 	 	& 	4.280E-18 & 	 	 & 	6.407E-15 \\ \hline 
+		 Cf249	 & 	 	& 	1.649E-12 & 	 	 & 	6.446E-10 \\ \hline 
+		 Cf250	 & 	 	& 	2.041E-12 & 	 	 & 	6.703E-11 \\ \hline 
+		 Cf251	 & 	 	& 	9.865E-13 & 	 	 & 	1.903E-12 \\ \hline 
+		 Cf 252	 & 	 	& 	6.579E-13 & 	 	 & 	4.014E-14 \\ \hline 
+		 H3	 & 	 	& 	8.584E-08 & 	 	 & 	1.747E-07 \\ \hline 
+		 C14	 & 	 	& 	4.057E-11 & 	 	 & 	 	 \\ \hline 
+		 C Other	 & 	 	& 	 	 & 	 	 & 	 	 \\ \hline 
+		 Kr81	 & 	 	& 	4.216E-11 & 	 	 & 	8.038E-12 \\ \hline 
+		 Kr85	 & 	 	& 	3.444E-05 & 	 	 & 	2.950E-05 \\ \hline 
+		 Kr Other	 & 	 	& 	0.000 & 	 	 & 	0.000 \\ \hline 
+		 Sr90	 & 	 	& 	0.001 & 	 	 & 	0.001 \\ \hline 
+		 Sr Other	 & 	 	& 	0.000 & 	 	 & 	0.000 \\ \hline 
+		 Tc99	 & 	 	& 	0.000 & 	 	 & 	5.391E-05 \\ \hline 
+		 Tc Other	 & 	 	& 	0.000 & 	 	 & 	0.002 \\ \hline 
+
+		\end{tabular}}
+		\caption{Fresh and Spent Fuel Compositions}
+		\label{tab:comp}
+\end {table}
+\end{frame}
+
+%tikz styling definitions
+
+\tikzstyle{decision} = [diamond, draw, fill=blue!20, 
+text width=4.5em, text badly centered, node distance=3cm, inner sep=0pt]
+\tikzstyle{block} = [rectangle, draw, fill=blue!20, 
+text width=5em, text centered, rounded corners, minimum height=4em]
+\tikzstyle{line} = [draw, -latex']
+\tikzstyle{cloud} = [draw, ellipse,fill=red!20, node distance=3cm,
+minimum height=2em]
+
+\subsection{Simulation Specifications}
```

Comment:
Specification is a long word meaning lots of things. I think "parameter" might be more typical for this meaning. Consider replacing specification with "parameter" everywhere? nearly everywhere?